packages/progress-indicator/test/lion-progress-indicator.test.js

```diff
@@ -1,23 +1,125 @@
+import { expect, fixture as _fixture } from '@open-wc/testing';
 
 import '@lion/progress-indicator/define';
 
+/**
+ * @typedef {import('../src/LionProgressIndicator').LionProgressIndicator} LionProgressIndicator
+ * @typedef {import('@lion/core').TemplateResult} TemplateResult
+ */
+
+const fixture = /** @type {(arg: TemplateResult) => Promise<LionProgressIndicator>} */ (_fixture);
+
 describe('lion-progress-indicator', () => {
-  describe('Accessibility', () => {
-    it('adds a label', async () => {
+  describe('indeterminate', async () => {
+    it('is indeterminate when has no value attribute', async () => {
       const el = await fixture(html` <lion-progress-indicator></lion-progress-indicator> `);
-      expect(el.getAttribute('aria-label')).to.equal('Loading');
+      expect(el.indeterminate).to.be.true;
     });
+  });
 
-    it('sets the right role', async () => {
-      const el = await fixture(html` <lion-progress-indicator></lion-progress-indicator> `);
-      expect(el.getAttribute('role')).to.equal('status');
+  describe('determinate', async () => {
+    it('is determinate when has value attribute', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="25" aria-label="foo"></lion-progress-indicator> `,
+      );
+      expect(el.indeterminate).to.be.false;
     });
 
-    it('sets aria-live to "polite"', async () => {
-      const el = await fixture(html` <lion-progress-indicator></lion-progress-indicator> `);
-      expect(el.getAttribute('aria-live')).to.equal('polite');
+    it('can update value', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="25" aria-label="foo"></lion-progress-indicator> `,
+      );
+      expect(el.getAttribute('aria-valuenow')).to.be.eq('25');
+      el.value = 30;
+      await el.updateComplete;
+      expect(el.getAttribute('aria-valuenow')).to.be.eq('30');
+    });
+
+    it('can update min', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="50" aria-label="foo"></lion-progress-indicator> `,
+      );
+      expect(el.getAttribute('aria-valuemin')).to.be.eq('0');
+      el.min = 30;
+      await el.updateComplete;
+      expect(el.getAttribute('aria-valuemin')).to.be.eq('30');
+    });
+
+    it('can update max', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="50" aria-label="foo"></lion-progress-indicator> `,
+      );
+      expect(el.getAttribute('aria-valuemax')).to.be.eq('100');
+      el.max = 70;
+      await el.updateComplete;
+      expect(el.getAttribute('aria-valuemax')).to.be.eq('70');
+    });
+
+    it('min & max limits value', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator value="150" aria-label="foo"></lion-progress-indicator> `,
+      );
+      // sets to default max: 100
+      expect(el.getAttribute('aria-valuenow')).to.be.eq('100');
```

Comment:
I think all `to.be.eq` in this test can be replaced with `to.equal` (since eq is deep equal (meant for objects/arrays) and 'be' is redundant)